kratos/tests/test_vtk_output_process.py

```diff
@@ -176,8 +251,10 @@ def ExecuteBasicVTKoutputProcessCheck(file_format = "ascii", dimension = "2D"):
     model_part = current_model.CreateModelPart(model_part_name)
     if dimension == "2D":
         SetupModelPart2D(model_part)
-    else:
+    elif dimension == "3D":
```

Comment:
Quad should probably not be lumped in the same variable as dimension. This makes the code confusing.

Reply:
It was the way to make the less modifications possible

Reply:
One sec

Reply:
I changed dimension for setup

Reply:
Thanks. That makes the variable naming less confusing.